pom.xml

```diff
@@ -134,6 +134,16 @@
               <arguments>run lint</arguments>
             </configuration>
           </execution>
+          <execution>
+            <id>npm test:ci</id>
+            <goals>
+              <goal>npm</goal>
+            </goals>
+            <phase>generate-resources</phase>
```

Comment:
Ok maven it's overkill... but we can use test phase for this ?

Reply:
Sure! I copy/paste the `npm lint` step but I do agree this `generate-resources` feels weird 🤔  let's go with test